themes/default/content/docs/reference/pulumi-yaml.md

```diff
@@ -48,3 +48,9 @@ It contains the following required and optional attributes:
         - `description`: (optional) a description for the config setting.
         - `default`: (optional) the default value of the config setting - which will be presented to the user as a default.
         - `secret`: (optional) if `true` indicates that this configration value should be marked as secret.
+
+- `plugins`: (optional) allows the overriding of plugin selection to make use of plugins not installed into the global plugin cache
+  - `providers`/`analyzers`/`languages`: (optional) Each entry has a list of plugins underneath it.
```

Comment:
```suggestion
  - `providers`/`analyzers`/`languages`: (optional) each entry has a list of plugins underneath it.
```

would only saying `list of plugins.` make sense?

Reply:
Yes, but given the slight odd format of this I was erring on a longer description so people trigger that this is multiple entries like:
```
plugins:
  providers:
  - name: aws
    path: ../aws
  analyzers:
  - name: company_policy
    path: ./policy
```